crates/lib/src/qpu/quilc/mod.rs

```diff
@@ -68,7 +74,7 @@ impl Error {
 /// A wrapper around a [`String`] which indicates the string contains valid Native Quil. That is,
 /// Quil which has been processed through `quilc`.
 #[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
-pub struct NativeQuil(String);
+pub struct NativeQuil(pub String);
```

Comment:
One possible concern here is that the now-public inner string can be modified to be invalid. If this is meant to guarantee it is valid, I'd leave it private and implement `AsRef<str>` and `From<NativeQuil> for String ` instead.